### ISSUE TITLE
Provide PS Repository name as parameter to each of the "Install-Module" commands

### ISSUE THREAD

**Issue by antonystefanov** [vmware/powershell-module-for-vmware-cloud-foundation-reporting#188]
### Code of Conduct

- [X] I have read and agree to the project's Code of Conduct.
- [X] Vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue initial description to help the maintainers prioritize.
- [X] Do not leave "+1" or other comments that do not add relevant information or questions.
- [X] If you are interested in working on this issue or have submitted a pull request, please leave a comment.

### Description

In the [installation instructions](https://vmware.github.io/powershell-module-for-vmware-cloud-foundation-reporting/install/) we provide commands:
```
Set-PSRepository -Name PSGallery -InstallationPolicy Trusted
Install-Module -Name VMware.PowerCLI -MinimumVersion 13.1.0
Install-Module -Name VMware.vSphere.SsoAdmin -MinimumVersion 1.3.9
Install-Module -Name PowerVCF -MinimumVersion 2.3.0
Install-Module -Name PowerValidatedSolutions -MinimumVersion 2.6.0
Install-Module -Name VMware.CloudFoundation.Reporting
```
In the first row we set the PSGallery PS Repository  `Set-PSRepository -Name PSGallery -InstallationPolicy Trusted` from where to get modules installed. In customer environment there could be multiple PS Repository set up. When try to install a modules there could be an error that same module and version is found in multiple PS Repositories, or that there is version missmatch across PS Repositories.
To avoid this we could provide PS Repository name as parameter to each of the "Install-Module" commands like this:

`Install-Module -Name VMware.PowerCLI -MinimumVersion 13.1.0 -PSrepository PSGallery`

### Use Case(s)

Installation of PS modules in environment where there are multiple PS Repositories configured.

### Potential Configuration

```
Set-PSRepository -Name PSGallery -InstallationPolicy Trusted
Install-Module -Name VMware.PowerCLI -MinimumVersion 13.1.0 -Repository PSGallery
Install-Module -Name VMware.vSphere.SsoAdmin -MinimumVersion 1.3.9 -Repository PSGallery
Install-Module -Name PowerVCF -MinimumVersion 2.3.0 -Repository PSGallery
Install-Module -Name PowerValidatedSolutions -MinimumVersion 2.6.0 -Repository PSGallery
Install-Module -Name VMware.CloudFoundation.Reporting -Repository PSGallery
```

**Comment by tenthirtyam** [comment]
@burnsjared0415 - could you pick this one up, by chance? Ty!

**Comment by burnsjared0415** [comment]
added to documentation and pushed to develop

**Comment by tenthirtyam** [comment]
Resolved in #189.
